ingredient_phrase_tagger/training/tokenizer.py

```diff
@@ -1,16 +1,25 @@
+#!/usr/bin/env python3
 import re
 
-import utils
+
+def clumpFractions(s):
+    """
+    Replaces the whitespace between the integer and fractional part of a quantity
+    with a dollar sign, so it's interpreted as a single token. The rest of the
+    string is left alone.
+
+        clumpFractions("aaa 1 2/3 bbb")
+        # => "aaa 1$2/3 bbb"
+    """
+    return re.sub(r'(\d+)\s+(\d)/(\d)', r'\1$\2/\3', s)
 
 
 def tokenize(s):
     """
     Tokenize on parenthesis, punctuation, spaces and American units followed by a slash.
-
     We sometimes give American units and metric units for baking recipes. For example:
         * 2 tablespoons/30 mililiters milk or cream
         * 2 1/2 cups/300 grams all-purpose flour
-
```

Comment:
Can we restore the blank lines on 9 and 13? They don't seem related to the 2-to-3 migration.